### PR TITLE
Milestone: Portfolio desktop version

### PR DESCRIPTION
**Pull request by ForHemer**
This milestone contains the desktop version of my personal portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,180 +5,183 @@
...
-                  <li>Capibara</li>
-                  <li>Selenium</li>
-              </ul>
-            </div>
-  
-          <div id="Skills">
-              <h3>Skills</h3>
-            <ul>
-                <li>Codekit</li>
-                <li>Codepen</li>
```
Comment (Hamzaoutdoors) [inline index.html:148]: - [x] Great work adding the desktop version to your project, but unfortunately it's not showing up two main sections of the project the `About me` & `Get Started` as you can see in the screenshots below, Kindly, fix this issue🙏🏼☺️.

I think you delete them in this working branch by mistake

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
  |

File: index.html [hunk 2]
```diff
@@ -5,180 +5,183 @@
...
-                <li><a href=""><img src="images/ic_stackoverflow.svg" alt="stackoverflow"></a></li>
-                <li><a href="https://github.com/ForHemer"><img src="images/ic_github.svg" alt="GitHub"></a></li>
-                <li><a href="http://twitter.com/HemersonForeste"><img src="images/ic_twitter.svg" alt="twitter"></a></li>
-                <li><a href="https://www.linkedin.com/in/hemerson-foreste-890685197/"><img src="images/ic_linkedin.svg" alt="Linkedin"></a></li>
-                <li><a href=""><img src="images/ic_medium.svg" alt="Medium"></a></li>
-                <li><a href=""><img src="images/ic_angellist.svg" alt="Angellist"></a></li>
-            </ul>
-        </div>
-        </div>
-      </section>
```
Comment (Hamzaoutdoors) [inline index.html:181]: - [x]  The same issue in this part the form section is not showing up as you can see in the screenshots below, Kindly, fix this issue🙏🏼☺️.

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
  |

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @ForHemer, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Good job respecting the desktop breakpoint 👍🏼 

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🔥 

### Hi @team 👋🏻, 

You did a tremendous job implementing the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**